app/components/SidebarLayout.tsx

```diff
@@ -12,6 +12,7 @@ import {
   FaGaugeHigh,
   FaLocationDot,
   FaAngleRight,
+  FaUserGroup,
```

Comment:
This is outside the scope of this PR (literally), but a couple of lines down is a typo for `Container` that would be awesome if you'd consider tagging along in this work.

Reply:
gruesome stuff—and now fixed. thank you!